### ISSUE TITLE
LogiX Node Request: User Grabber

### ISSUE THREAD

**Issue by orange3134** [Neos-Metaverse/NeosPublic#3160]
### Is your feature request related to a problem? Please describe.

No, I just found it useful to have this node.

### Describe the solution you'd like

I want a node to get the Grabber of User.
This node has User and left / right inputs.
And it outputs Grabber.

### Describe alternatives you've considered

Find the Grabber Slot by combining several nodes.
It's not impossible, but it's a bit of a hassle because each avatar has a different structure.

**Comment by H3BO3** [comment]
I agree with this feature - I have to do a strange enum setup to pipe the active user into standard controller and then provide it with the right chirality as well, when it would just be easier to have those events/bools on a single node.

**Comment by rheniumNV** [comment]
I too would like to see this feature implemented.

Currently I am trying to create a tool that will open the object in my hand in the inspector. However, it is difficult to get the object in my hand in LogiX. This is because each avatar has a different structure.
I think this problem will be solved once this node is implemented.
